diff --git a/index.js b/index.js
index 4e8423491ab51a9eb9fee22182e4ea0fcc3d3d3b..fc0bd347331fcbdd1128f2b5b6e3d2fb3e2ef5c5 100644
--- a/index.js
+++ b/index.js
@@ -1,12 +1,9 @@
 "use strict";
 
-const { load, currentTarget } = require("@neon-rs/load");
+const { currentTarget } = require("@neon-rs/load");
 const { familySync, GLIBC } = require("detect-libc");
 
 function requireNative() {
-  if (process.env.LIBSQL_JS_DEV) {
-    return load(__dirname)
-  }
   let target = currentTarget();
   // Workaround for Bun, which reports a musl target, but really wants glibc...
   if (familySync() == GLIBC) {
@@ -114,7 +111,7 @@ class Database {
   prepare(sql) {
     try {
       const stmt = databasePrepareSync.call(this.db, sql);
-      return new Statement(stmt);  
+      return new Statement(stmt);
     } catch (err) {
       throw convertError(err);
     }
@@ -296,7 +293,7 @@ class Statement {
         return statementRun.call(this.stmt, bindParameters[0]);
       } else {
         return statementRun.call(this.stmt, bindParameters.flat());
-      }  
+      }
     } catch (err) {
       throw convertError(err);
     }
