diff --git a/node_modules/libsql/.bun-tag-8672db3ec580bbb6 b/.bun-tag-8672db3ec580bbb6
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/index.js b/index.js
index 4e8423491ab51a9eb9fee22182e4ea0fcc3d3d3b..a89e10295bc7ff2869bd97728427e5b70e6b0ca7 100644
--- a/index.js
+++ b/index.js
@@ -19,7 +19,7 @@ function requireNative() {
       break;
     }
   }
-  return require(`@libsql/${target}`);
+  return require(`../@libsql/${target}/index.node`);
 }
 
 const {
@@ -114,7 +114,7 @@ class Database {
   prepare(sql) {
     try {
       const stmt = databasePrepareSync.call(this.db, sql);
-      return new Statement(stmt);  
+      return new Statement(stmt);
     } catch (err) {
       throw convertError(err);
     }
@@ -296,7 +296,7 @@ class Statement {
         return statementRun.call(this.stmt, bindParameters[0]);
       } else {
         return statementRun.call(this.stmt, bindParameters.flat());
-      }  
+      }
     } catch (err) {
       throw convertError(err);
     }
